mypy-nofollow.ini

```diff
@@ -23,6 +23,9 @@ files =
 # in Python 3.8
 python_version = 3.8
 
+[mypy-deeplearning.*]
+ignore_missing_imports = True
```

Comment:
what is mypy-deeplearning ? i dont follow.

Reply:
I think it's one of our internal libraries. This setting ensures mypy doesn't raise an error on imports like

https://github.com/pytorch/pytorch/blob/a0e9dcec4c883d04d36a66328d8ecc4480f8027d/torch/_inductor/fx_passes/group_batch_fusion.py#L18

because it can't see the imported source files.